src/rust/src/backend/utils.rs

```diff
@@ -45,6 +46,7 @@ pub(crate) fn pkey_private_bytes<'p>(
     format: &pyo3::PyAny,
     encryption_algorithm: &pyo3::PyAny,
     openssh_allowed: bool,
+    raw_allowed: bool,
```

Comment:
This is an ugly function signature, although I don't have a lot of ideas on how to make it better.

Reply:
Dislike the various `_allowed`? We could add a `SerializationConfig` with fields for each of them. Seems like only a marginal improvement though.

Reply:
Yeah I don't like the alloweds, but I agree that a struct probably isn't much better here.